Can't use `useSubspace` from subspace-react in typescript projects

Reported by @acolytec3 :
> Usage question on useSubspace from subspace-react, if I say import { useSubspace } from '@embarklabs/subspace-react' in a typescript project, Typescript complains that it can't find the method.  If I change it to import useSubspace from '@embarklabs/subspace-react/types the app refuses to compile saying Module not found: Can't resolve '@embarklabs/subspace-react/types'  in ../home/{project_directory}/{component}. Shouldn't the compiler be able to resolve the module since it sees it in the module tree?

>I also tried npm install --save @types/subspace-react and it says the types aren't in the types-registry and then set up a @types/subspace-react dependency in package.json that points to the local node-modules/subspace-react directory and npm installed that without complaint but says modulenotfound and won't compile when I do import useSubspace from 'subspace-react

--- a/packages/react/types/index.d.ts
+++ b/packages/react/types/index.d.ts
@@ -12,7 +12,7 @@ export const SubspaceProvider: React.Component<SubspaceProviderProps>;
 
 export class observe extends React.Component<P> {}
 
-export default function useSubspace(): Subspace;
+export function useSubspace(): Subspace;
 
 interface withSubspaceProps {
   subspace: Subspace;

